domains/seiko.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Delta-0mega",
+    "email": "delta-omegaone@proton.me"
+  },
+  "record": {
+    "A": [
+      "38.22.104.142"
```

Comment:
Nginx Preview page. Can you send a preview of the actual stie?

Reply:
I think this is normal. I did the same thing with another domain and it put me on the right page. This is the way micium-hosting uses to connect domain names.

Reply:
![Screenshot_2024-10-06-09-36-26-517_com.brave.browser.jpg](https://github.com/user-attachments/assets/8f920cc9-a2a7-4a50-acd0-2434ba952bef)

Reply:
![Screenshot_2024-10-06-09-37-51-082_com.discord-edit.jpg](https://github.com/user-attachments/assets/5b90f8be-025e-46c4-95a0-60f79478a1cb)